### PR TITLE
LL-4579 (Theme): track os theme and change app theme accordingly

### PR DESCRIPTION
**Pull request by LFBarreto**
(Theme): track os theme and change app theme accordingly
### Type

UI Polish
### Context

[LL-4579]
### Parts of the app affected / Test plan

Open the app and switch Os theme, reopenning app or reentering it should trigger a theme update if needed.

[LL-4579]: https://ledgerhq.atlassian.net/browse/LL-4579

### CODE REVIEW

File: src/reducers/settings.js [hunk 1]
```diff
@@ -293,11 +295,15 @@ const handlers: Object = {
...
+  SETTINGS_SET_OS_THEME: (state, { payload: osTheme }) => ({
+    ...state,
+    osTheme,
+  }),
+  SETTINGS_SET_CAROUSEL_VISIBILITY: (state: SettingsState, { payload }) => ({
     ...state,
     carouselVisibility: payload,
   }),
-  SETTINGS_SET_DISCREET_MODE: (state: AppState, { payload }) => ({
+  SETTINGS_SET_DISCREET_MODE: (state: SettingsState, { payload }) => ({
```
Comment (juan-cortes) [inline src/reducers/settings.js:306]: Yeah AppState was by bad, good catch

**Review by juan-cortes — DISMISSED** [review]
lgtm

**Review by Arnaud97234 — COMMENTED** [review]
On iOS the theme doesn't switch corresponding to the OS theme.

File: app.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
 {
   "name": "ledgerlivemobile",
-  "displayName": "Ledger Live"
+  "displayName": "Ledger Live",
+  "userInterfaceStyle": "automatic"
```
Comment (gre) [inline app.json:4]: wow what is this magic

**Review by Arnaud97234 — APPROVED** [review]

**Review by gre — APPROVED** [review]

**Pull request merged**